### ISSUE TITLE
[Feature Request]: ARM64 Support for Linux

### ISSUE THREAD

**Issue by yg5757** [ThaUnknown/miru#184]
### Preflight checklist

- [ ] I have searched the [issue tracker](https://github.com/ThaUnknown/miru/issues) for a bug report that matches the one I want to file, without success.
- [x] I have searched the [features list](https://github.com/ThaUnknown/miru#features) for this feature, and I couldn't find it.
- [ ] I have checked that I'm using the [latest stable](https://github.com/ThaUnknown/miru/releases/latest) version of the app.

### Problem Description

It would be nice to have a build that supports ARM64. I'm using linux with an arm based cpu, and can't install Miru.

### Proposed Solution

Support ARM based processors.

**Comment by ThaUnknown** [comment]
as of now u can just build it locally if you want to

**Comment by ThaUnknown** [comment]
there's a problem with this:

TLDR we compiled a custom version of chromium to support more video codecs, etc etc but we don't own a linux ARM based device to compile that specific version of chromium for, so for this to happen we'd need you to compile this custom version of electron for 18.2.1: https://github.com/Muril-o/electron-chromium-codecs before we can really support this

**Comment by yg5757** [comment]
I'm not that experienced with Linux, so I wouldn't know how do to that sadly.

**Comment by ThaUnknown** [comment]
cc20e35500522715483a73e2bbb5c4d23010e573
but it doesn't include the custom build with codecs, so cope without HEVC, AC3 and EAC3
